src/initializers/riviere.ts

```diff
@@ -45,6 +45,7 @@ const init = (config, orkaOptions) => {
     styles: config.riviere.styles,
     bodyKeys: config.riviere.bodyKeys,
     bodyKeysRegex: config.riviere.bodyKeysRegex && new RegExp(config.riviere.bodyKeysRegex, 'i'),
+    bodyKeysCallback: config.riviere.bodyKeysCallback,
```

Comment:
🙃 Is this configuration included in docs? Shall we add it?

Reply:
We don't explicitly defining `riviere` configuration on `orka's` docs. I've included it to `riviere` configuration though